tools/tox_matrix.py

```diff
@@ -138,6 +140,8 @@ def get_matrix_item(env, global_libraries, global_string_parameters,
         sep = r"\\" if platform == "windows" else "/"
```

Comment:
This will need to be before the if block now

Reply:
thanks! Sorry for all the changes 😅